AppAuth-JS/built/revoke_token_request.js

```diff
@@ -0,0 +1,50 @@
+"use strict";
```

Comment:
This is generated source file. All the files in built/ folder are generated JS file for the Google's lib fork